go-controller/pkg/ovn/policy_test.go

```diff
@@ -542,7 +548,8 @@ var _ = Describe("OVN NetworkPolicy Operations", func() {
 				nTest.baseCmds(fExec, namespace1, namespace2)
 				nTest.addCmds(fExec, namespace2)
 				nTest.addCmdsWithPods(fExec, nPodTest, namespace1)
-				npTest.addPodSelectorCmds(fExec, nPodTest, networkPolicy, true, true)
+				npTest.addNamespaceSelectorCmds(fExec, networkPolicy, true)
```

Comment:
This test looked wrong; addPodSelectorCmds() is supposed to be used for NPs that have a `podSelector` which this test is not exercising (it only has a `namespaceSelector`).